lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -732,12 +738,13 @@ private void processStandardSettingsForConfiguration(TaskImpl task, Classificati
         if (task.getDescription() == null && classification != null) {
             task.setDescription(classification.getDescription());
         }
+        LOGGER.debug("exit from processStandardSettingsForConfiguration()");
     }
 
     private BulkOperationResults<String, TaskanaException> transferTasks(List<String> taskIdsToBeTransferred,
         Workbasket destinationWorkbasket)
         throws InvalidArgumentException, WorkbasketNotFoundException, NotAuthorizedException {
-
+        LOGGER.debug("entry to transferTasks(taskIdsToBeTransferred = {}, destinationWorkbasket = {})", LoggerUtils.listToString(taskIdsToBeTransferred), destinationWorkbasket);
         workbasketService.checkAuthorization(destinationWorkbasket.getId(), WorkbasketPermission.APPEND);
```

Comment:
The LOGGER statement on line 747 should only be called if debug is enabled (to invoke listToString() only when required...